onomy/chain.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "onomy",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://onomy.io/",
+  "pretty_name": "Onomy",
+  "chain_id": "onomy-mainnet-1",
+  "bech32_prefix": "onomy",
```

Comment:
Please register this prefix to SLIP-0173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Submitted PR

Reply:
https://github.com/satoshilabs/slips/pull/1477